atlas/emails/finished.vm

```diff
@@ -0,0 +1,10 @@
+Hello,
```

Comment:
with a file with .vm extension ? template emails are using .tpl extension in extractorapp & ldapadmin